### PR TITLE
added Readme

### PR DESCRIPTION
**Pull request by ThijsSassen**
added readme with known limitations of Vortex OpenSplice

### CODE REVIEW

**Comment by dirk-thomas** [comment]
> Once sentence per line.

The [guidelines](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#markdown-restructured-text-docblocks) don't say "don't wrap within one sentence". They say each sentence needs to start on a new line and the current patch follows that.

**Comment by mjcarroll** [comment]
Whoops, you are correct @dirk-thomas, sorry about that.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# A ROS2 RMW implementation for Vortex OpenSplice
```
Comment (dirk-thomas) [inline README.md]: Please add a space between "ROS" and "2".

Same below.

Comment (ThijsSassen) [reply README.md]: ok done

**Comment by dirk-thomas** [comment]
Thanks.

**Comment by dirk-thomas** [comment]
For cross referencing: related to #292.

**Pull request merged**